backend/src/test/java/ai/verta/modeldb/ExperimentTest.java

```diff
@@ -1865,4 +1865,42 @@ public void deleteExperiments() throws InterruptedException {
 
     LOGGER.info("Batch Delete Experiment test stop................................");
   }
+
+  @Test
+  public void updateExperimentName() {
+    LOGGER.info("Update Experiment Name test start................................");
+
+    Experiment experiment2 = null;
+    try {
+      CreateExperiment createExperimentRequest =
+          getCreateExperimentRequest(project.getId(), "Experiment-" + new Date().getTime());
+      CreateExperiment.Response createExperimentResponse =
+          experimentServiceStub.createExperiment(createExperimentRequest);
+      experiment2 = createExperimentResponse.getExperiment();
+
+      UpdateExperimentName updateNameRequest =
+          UpdateExperimentName.newBuilder()
+              .setId(experiment.getId())
+              .setName(experiment2.getName())
+              .build();
+
+      experimentServiceStub.updateExperimentName(updateNameRequest);
+      fail();
+    } catch (StatusRuntimeException ex) {
+      Status status = Status.fromThrowable(ex);
+      LOGGER.warn("Error Code : " + status.getCode() + " Description : " + status.getDescription());
```

Comment:
This error is expected as part of the test. Info at most. 
```suggestion
      LOGGER.info("Error Code : " + status.getCode() + " Description : " + status.getDescription());
```